### PR TITLE
Clinician tests refactoring/clean up

### PR DESCRIPTION
**Pull request by nmajor25**
## Summary by CodeRabbit

- **Bug Fixes**
	- Enhanced integration tests for the clinicians modal by replacing hardcoded IDs with dynamic references to current role and team data.
	- Updated clinician sidebar tests to use dynamically generated UUIDs for clinician IDs, improving test reliability.
	- Modified clinicians list tests to utilize UUIDs for clinician IDs, ensuring uniqueness and reducing conflicts.

- **Chores**
	- Removed unnecessary attributes from test data to simplify the testing framework.

### CODE REVIEW

File: test/integration/clinicians/clinician-sidebar.js [hunk 1]
```diff
@@ -1,24 +1,22 @@
...
 
 import { teamCoordinator, teamNurse } from 'support/api/teams';
 import { roleAdmin, roleEmployee, roleManager } from 'support/api/roles';
-import { getWorkspaces, workspaceOne } from 'support/api/workspaces';
+import { workspaceOne } from 'support/api/workspaces';
 import { getClinician, getCurrentClinician } from 'support/api/clinicians';
 
 const testClinician = getClinician({
-  id: '1',
+  id: uuid(),
```
Comment (nmajor25) [inline test/integration/clinicians/clinician-sidebar.js:13]: I can't remember why it's required to include an `id` in `getClinician()`... is it so we don't interfere with the current clinician?

File: test/integration/clinicians/clinician-sidebar.js [hunk 2]
```diff
@@ -1,24 +1,22 @@
...
   attributes: {
     name: 'Test Clinician',
     email: 'test.clinician@roundingwell.com',
-    last_active_at: testTs(),
-    enabled: true,
```
Comment (nmajor25) [inline test/integration/clinicians/clinician-sidebar.js:17]: `enabled: true` is set for each clinician fixture, so we don't need to set it in `getClinician()`.

File: test/integration/clinicians/clinician-sidebar.js (hunk 1 above)
Comment (paulfalgout) [reply test/integration/clinicians/clinician-sidebar.js:13]: well the trick with clinicians is that they're assigned to global things.. so if you change the uuid here for just this instance then it won't necessarily be available in the owner component droplist..  so we need a "there's X number of test clinicians that work everywhere and when I modify it, it's going to be modified everywhere". I think the clinician part of the app is the only part where we can use generated `uuid`s without something else breaking elsewhere.

Comment (nmajor25) [reply test/integration/clinicians/clinician-sidebar.js:13]: Gotcha. So should we keep this as `id: '1'` for now? Or maybe switch it to `22222` or something until we update it later?

**Comment by paulfalgout** [comment]
Can you rebase this off of develop?

**Comment by nmajor25** [comment]
@paulfalgout yep, rebased 👍

**Comment by nmajor25** [comment]
@paulfalgout what is there still to do for this one?

**Comment by paulfalgout** [comment]
> @paulfalgout what is there still to do for this one?

well I suppose we can get this in, but we should go ahead and work to remove the need to send an id to `getClinician` using the same patterns as the workspace/program work of pulling the first n fixtures needed for global entity testing.

**Review by paulfalgout — APPROVED** [review]

**Comment by nmajor25** [comment]
@paulfalgout ah ya, okay I'll create a shortcut story for that

**Pull request merged**